spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/src/test/java/com/example/DatastorePersonRepository.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import java.util.List;
+
+import org.springframework.cloud.gcp.data.datastore.repository.DatastoreRepository;
+import org.springframework.cloud.gcp.data.datastore.repository.query.Query;
+import org.springframework.data.repository.query.Param;
+
+/**
+ * Sample repository for testing.
+ *
+ * @author Chengyuan Zhao
+ */
+public interface DatastorePersonRepository extends DatastoreRepository<Person, Long> {
```

Comment:
Again, I would avoid any Datastore deps in the Spanner sample.

Reply:
done